### PR TITLE
Log the size of the data sent in the out_oms_blob.rb

### PR DESCRIPTION
**Pull request by robbiezhang**
Log the size of the data sent in the out_oms_blob.rb
@Microsoft/omsagent-devs @agup006

### CODE REVIEW

File: source/code/plugins/out_oms_blob.rb [hunk 1]
```diff
@@ -115,12 +115,17 @@ def get_blob_uri(container_type, data_type, custom_data_type, suffix)
     #   msgs: string[]. messages
     def append_blob(uri, msgs)
       if msgs.size == 0
-        return true
+        return 0
```
Comment (jeffaco) [inline source/code/plugins/out_oms_blob.rb]: 'return false' for consistency?

File: source/code/plugins/out_oms_blob.rb [hunk 2]
```diff
@@ -227,9 +233,9 @@ def handle_records(tag, records)
       @log.debug "Success getting the BLOB uri in #{time.round(3)}s"
 
       start = Time.now
-      append_blob(blob_uri, records)
+      dataSize = append_blob(blob_uri, records)
       time = Time.now - start
-      @log.debug "Success sending the data to BLOB #{time.round(3)}s"
+      @log.debug "Success sending #{dataSize} bytes data to BLOB #{time.round(3)}s"
```
Comment (jeffaco) [inline source/code/plugins/out_oms_blob.rb]: Success sending #{dataSize} bytes **of** data ...

Insert "of" for grammatical correctness

**Comment by jeffaco** [comment]
Signed off

File: source/code/plugins/out_oms_blob.rb [hunk 3]
```diff
@@ -115,12 +115,17 @@ def get_blob_uri(container_type, data_type, custom_data_type, suffix)
...
     def append_blob(uri, msgs)
       if msgs.size == 0
-        return true
+        return 0
       end
 
       # concatenate the messages
       msg = ''
       msgs.each { |s| msg << "#{s}\r\n" if s.to_s.length > 0 }
+      dataSize = msg.to_s.length
```
Comment (niroyb) [inline source/code/plugins/out_oms_blob.rb]: `msg` is already a string

**Comment by niroyb** [comment]
Signed off

File: source/code/plugins/out_oms_blob.rb (hunk 1 above)
Comment (robbiezhang) [inline source/code/plugins/out_oms_blob.rb]: the return value was not used before. I now change it to return the size of the data.

**Pull request merged**